calibrators/mapping_based_calibrator/README.md

```diff
@@ -0,0 +1,238 @@
+# mapping_based_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/mapping_based_calibrator.md)
+
+## Purpose
+
+The package `mapping_based_calibrator` allows extrinsic calibration between 3d lidar and 3d lidar sensors, as well as baselink and 3d lidar sensors used in autonomous driving and robotics.
```

Comment:
baselink?

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/9d68e7a7bb19858191090e5dc7658d92883fc208

Reply:
There is an additional () in the commit, I removed it in the https://github.com/tier4/CalibrationTools/pull/164/commits/2a1aa92d2ae84f61feefc15d79b77fc118e784cd